src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -66,17 +102,33 @@ public boolean isArcadeDrive() {
   }
 
   public void stopMotors() {
+    //We call .arcadeDrive with a speed and rotation of zero in hopes to command a zero output to the motors
     mDiffDrive.arcadeDrive(0.0, 0.0);
   }
 
   @Override
-  public void periodic() {
-    SmartDashboard.putNumber("Left Front Encoder Position", mLeftFront.getEncoder().getPosition());
-    SmartDashboard.putNumber("Right Front Encoder Position", mRightFront.getEncoder().getPosition());
-    SmartDashboard.putNumber("Left Front Encoder Velocity", mLeftFront.getEncoder().getVelocity());
-    SmartDashboard.putNumber("Right Front Encoder Velocity", mRightFront.getEncoder().getVelocity());
+  public void periodic()
+  {
+    //The simulation crashes whenever .getEncoder() is called
+    if (!Robot.isSimulation())
+    {
+      SmartDashboard.putNumber("Differential Right Front Encoder Position", mRightFrontEncoder.getPosition());
+      SmartDashboard.putNumber("Differential Right Middle Encoder Position", mRightMiddleEncoder.getPosition());
+      SmartDashboard.putNumber("Differential Right Rear Encoder Position", mRightRearEncoder.getPosition());
+      SmartDashboard.putNumber("Differential Left Front Encoder Position", mLeftFrontEncoder.getPosition());
+      SmartDashboard.putNumber("Differential Left Middle Encoder Position", mLeftMiddleEncoder.getPosition());
+      SmartDashboard.putNumber("Differential Left Rear Encoder Position", mLeftRearEncoder.getPosition());
+      SmartDashboard.putNumber("Differential Right Front Encoder Velocity", mRightFrontEncoder.getVelocity());
+      SmartDashboard.putNumber("Differential Right Middle Encoder Velocity", mRightMiddleEncoder.getVelocity());
+      SmartDashboard.putNumber("Differential Right Rear Encoder Velocity", mRightRearEncoder.getVelocity());
+      SmartDashboard.putNumber("Differential Left Front Encoder Velocity", mLeftFrontEncoder.getVelocity());
+      SmartDashboard.putNumber("Differential Left Middle Encoder Velocity", mLeftMiddleEncoder.getVelocity());
+      SmartDashboard.putNumber("Differential Left Rear Encoder Velocity", mLeftRearEncoder.getVelocity());
```

Comment:
As above it could be made simpler if the CANEncoders were an array that could be iterated over.

Reply:
Created a new issue #9 to track, because I don't think this should block the PR and should be a future upgrade instead.